### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by rqpt** [adalessa/laravel.nvim#98] [General]
Hi, I've been digging deeper into this plugin and I have to say - awesome work man.

The tooling around laravel in neovim, I'm sure you'll agree, is a bit lacking when compared with something like what phpstorm offers. This is one of those plugins that really carries the editor and makes it more than viable for laravel development.

Just wanted to say thanks - I pray this project has a long life ahead.

**Comment by adalessa** [comment]
Hey, thank you very much for the kind words I really appreciate them. :smile:
